learning_journal/views.py

```diff
@@ -1,36 +1,48 @@
-from pyramid.response import Response
 from pyramid.view import view_config
-
-from sqlalchemy.exc import DBAPIError
-
+from wtforms import Form, BooleanField, StringField, validators
+import pyramid.httpexceptions as ex
+import transaction
 from .models import (
     DBSession,
-    MyModel,
-    )
+    Entry,
+    NewEntry,
+)
+
+
+@view_config(route_name='home', renderer='templates/list.jinja2')
+def list_view(request):
+    entries = DBSession.query(Entry).order_by(Entry.created.desc()).all()
```

Comment:
If you return the query object instead of calling `.all()` you have more flexibility to use this as the basis of other view queries.  As it is now, you end up with a list of entries rather than a query.  You can't further modify that list using SQL.